controller/internal/windows/frontman/driver_windows.go

```diff
@@ -3,10 +3,236 @@
 package frontman
 
 import (
-	"fmt"
 	"syscall"
 )
 
+// ABI represents the 'application binary interface' to the Frontman dll
+type ABI interface {
```

Comment:
To me ABI is too broad of a name, because it could apply to any dll interface.  I think it should be something like Frontman, WindowNetDriver,  or WindowsPacketDriver.

Reply:
My thought was, since it's in the 'frontman' package, it would be referred to as 'frontman.ABI' which would make it more clear.

Reply:
Ok